extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -69,20 +71,20 @@ class AmpStickyAd extends AMP.BaseElement {
 
   /** @override */
   unlayoutCallback() {
+    this.viewport_.updatePaddingBottom(0);
     return true;
   }
 
   /** @override */
   detachedCallback() {
+    this.viewport_.updatePaddingBottom(0);
```

Comment:
The only place I add it previously is inside closeButton onClick function. I can't really think of other places where this would be detached, if that's the case I agree this is indeed not necessary.